the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  SettingsTableViewController.swift
+//  The Blue Alliance
+//
+//  Created by Anas Merbouh on 18-05-20.
+//  Copyright © 2018 The Blue Alliance. All rights reserved.
+//
+
+import UIKit
+
+class SettingsTableViewController: UITableViewController {
+    
+    // MARK: - View's lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "BaseCell")
+        tableView.register(UINib(nibName: "SwitchTableViewCell", bundle: nil), forCellReuseIdentifier: "SwitchCell")
+    }
+    
+    // MARK: - Interface Methods
+    
+    @IBAction func doneTapped(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: - Table view configuration
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 4
```

Comment:
Can we make this a struct and have a max number? I think we do this sort of thing in Team Info https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/swift/the-blue-alliance-ios/View%20Controllers/Teams/TeamInfoTableViewController.swift#L5